### ISSUE TITLE
Package request: shikane

### ISSUE THREAD

**Issue by emrebicer** [NixOS/nixpkgs#229161]
**Project description**

A dynamic output configuration tool for wlroots that automatically detects and configures connected outputs based on a set of profiles.

**Metadata**

* homepage URL: https://gitlab.com/w0lff/shikane
* source URL: https://gitlab.com/w0lff/shikane
* license: mit
* platforms: linux

**Comment by MichaelPachec0** [comment]
I already got the package config made in my config, will see about getting it into nixpkgs tomorrow https://github.com/MichaelPachec0/nix-config/blob/master/pkgs/shikane/default.nix

**Comment by natsukium** [comment]
@MichaelPachec0 
Sorry, I didn't notice your post and submitted a PR.
Could you please check it?
Also, would you like to become a maintainer?

**Comment by MichaelPachec0** [comment]
> @MichaelPachec0 Sorry, I didn't notice your post and submitted a PR. Could you please check it?
No worries, I will get around checking it today.
> Also, would you like to become a maintainer?
Yeah I would not mind doing so, it was partially why I took a stab at writing a pkg in the first place.

**Comment by MichaelPachec0** [comment]
@natsukium just to update, everything works.

**Comment by natsukium** [comment]
Sorry, I'm late.
Thanks for your confirmation.
Since it seems difficult to edit this branch with you, could you please add yourself to the maintainer-list in another PR?
Once it is merged, I will add you to the `shikane` maintainers.

**Comment by MichaelPachec0** [comment]
> Sorry, I'm late. Thanks for your confirmation. Since it seems difficult to edit this branch with you, could you please add yourself to the maintainer-list in another PR? Once it is merged, I will add you to the `shikane` maintainers.
Done #230969 will be waiting for the merge.
